cmd/skaffold/app/flags/image.go

```diff
@@ -61,9 +56,16 @@ func (i *Images) Set(value string) error {
 	return nil
 }
 
-// Type Implements Type() method for pflag interface
-func (i *Images) Type() string {
-	return fmt.Sprintf("%T", i)
+// Replace Implements Replace() method for pflag SliceValue interface
+func (i *Images) Replace(images []string) error {
+	newImages := NewEmptyImages("new list")
+	for _, value := range images {
+		a, err := convertImageToArtifact(value)
+		if err != nil {
+			return err
+		}
+		newImages.images = append(newImages.images, image{name: value, artifact: a})
+	}
 }
```

Comment:
I don't know that we need to use `NewEmptyImages()` here. I think we can just use `newImages := make([]image)`, then iterate over images like you've done and populate newImages, then do something like `i.images = newImages` to replace the underlying data

Reply:
Got it :+1